auth-db/init.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE auth (
+    id SERIAL PRIMARY KEY,
+    email TEXT,
+    password TEXT
```

Comment:
If this is hashed, isn’t there a constant size?

Reply:
Good point. I'll leave this for now, then update when I've worked out what hashing we're using 😄